staff-registration.php

```diff
@@ -0,0 +1,104 @@
+        <div class="row">
+            <div class="col-12 col-md-8 col-lg-8 offset-md-2 offset-lg-2">
+                <div class="row">
+                    <div class="col-12">
+                        <center>
+                            <h4>Sign Up - Staff</h4>
+                        </center>
+                    </div>
+                    <div class="col-12">
+                        <hr>
+                    </div>
+                    <div class="col-12 mt-3">
+                        <div class="row">
+                            <div class="col-12 col-md-4 col-lg-4">
+                                <label>First Name</label>
+                            </div>
+                            <div class="col-12 col-md-8 col-lg-8">
+                                <input type="text" class="staff-registration-input" id="fName">
+                            </div>
+                        </div>
+                    </div>
+                    <div class="col-12 mt-3">
+                        <div class="row">
+                            <div class="col-12 col-md-4 col-lg-4">
+                                <label>Last Name</label>
+                            </div>
+                            <div class="col-12 col-md-8 col-lg-8">
+                                <input type="text" class="staff-registration-input" id="lName">
+                            </div>
+                        </div>
+                    </div>
+                    <div class="col-12 mt-3">
+                        <div class="row">
+                            <div class="col-12 col-md-4 col-lg-4">
+                                <label>Email</label>
+                            </div>
+                            <div class="col-12 col-md-8 col-lg-8">
+                                <input type="email" class="staff-registration-input" id="email">
+                            </div>
+                        </div>
+                    </div>
+                    <div class="col-12 mt-3">
+                        <div class="row">
+                            <div class="col-12 col-md-4 col-lg-4">
+                                <label>Password</label>
+                            </div>
+                            <div class="col-12 col-md-8 col-lg-8">
+                                <input type="password" class="staff-registration-input" id="password">
+                            </div>
+                        </div>
+                    </div>
+                    <div class="col-12 mt-3">
+                        <div class="row">
+                            <div class="col-12 col-md-4 col-lg-4">
+                                <label>Role</label>
+                            </div>
+                            <div class="col-12 col-md-8 col-lg-8">
+                                <select name="role" id="role" class="staff-registration-input">
+                                    <option value="" selected>Select here</option>
```

Comment:
The value tag is empty. Add a proper value to the "value" tag

Reply:
OK